momento-redis/momento_redis.go

```diff
@@ -0,0 +1,53 @@
+package momento_redis
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+
+	"github.com/momentohq/client-sdk-go/momento"
+)
+
+// MomentoRedisClient wrapper over momento cache client that provides Redis compatible APIs
+type MomentoRedisClient struct {
+	client    momento.CacheClient
+	cacheName string
+}
+
+func NewMomentoRedisClient(cacheClient momento.CacheClient, cacheName string) *MomentoRedisClient {
+	client := MomentoRedisClient{
+		cacheName: cacheName,
+		client:    cacheClient,
+	}
+	return &client
+}
+
+func (c *MomentoRedisClient) String() string {
```

Comment:
Please rename this receiver to `m` to match the other receivers of this type in the `momento_redis` package. My IDE called this one out . . . I would have never noticed that on my own :-)